### ISSUE TITLE
Concatenation order of input for LSTM baseline. 

### ISSUE THREAD

**Issue by alexanderimanicowenrivers** [AI-ON/Few-Shot-Music-Generation#26]
I noticed that in the first baseline, you concat support + query (in that order), so I did the same. However, in the proposal, it seems to ask for query + support.

**Comment by sachinravi14** [comment]
For the first baseline, there is no difference between support and query sets so we are just creating a batch with all the songs and then training on them? So the concatenation is along the batch dimension just to create a batch of songs where order doesn't matter.

For the second baseline, I believe the concatenation is along the sequence dimension (rather than batch dimension) so the order does matter. But, I still believe it is support (or training set) first, followed by query (or test set)?

**Comment by larocheh** [comment]
Yes, I confirm what Sachin is saying.

**Comment by alexanderimanicowenrivers** [comment]
Hey,

Ahh I see. Then everything is correct and eval2, which I commited, does indeed evaluate the second baseline.

I will add in a flag tomorrow so that it can be selected using a script. 

Note, I wanted to make a seperate file named LSTM baseline 2 - however I don’t have the rights on GitHub to add files. 

Bests,
Alex 

Bests,
Alex

**Comment by sachinravi14** [comment]
You cannot add files in your own pull request? I think it should be defined in a separate file in `src/models/` and with configuration information in a `.yaml` file in `config/`

**Comment by alexanderimanicowenrivers** [comment]
All good now - I can create files. I have withdrawn my pull request for adding in eval2 and have instead added a separate file for the second baseline, which I am creating the config file for now.
